### PR TITLE
Add usage page to track current user stats

### PR DESCRIPTION
**Pull request by duvld**
## Description

Allow user to check the following stats on the new route `/account/usage`:
- Monthly submissions
- Total storage (in GB)
- Monthly transcription minutes
- Monthly translation characters

## Related issues

Part of billing

### CODE REVIEW

File: webpack/dev.server.js
Comment (bufke) [inline webpack/dev.server.js:9]: Did you intend to commit this?

File: jsapp/js/account/usage.api.ts
Comment (bufke) [inline jsapp/js/account/usage.api.ts:8]: Use unknown instead of any. Then if we end up using this later, TS will error and we can substitute in the right value. With any, it won't error and we don't think to go back and update it.

File: jsapp/js/account/usage.component.tsx
Comment (bufke) [inline jsapp/js/account/usage.component.tsx:55]: I'm pretty sure that @JacquelineMorrissette told me this isn't true. I think I just made this up in my head one day.

**Review by bufke — CHANGES_REQUESTED** [review]
Looks good otherwise. I'm not qualified to review CSS, maybe see if someone else can. Approved to merge pending the requested changes

File: jsapp/js/account/usage.component.tsx
Comment (JacquelineMorrissette) [reply jsapp/js/account/usage.component.tsx:55]: Can confirm, all our counters are updated in real time :+1:

File: webpack/dev.server.js
Comment (duvld) [reply webpack/dev.server.js:9]: Oops!

**Review by bufke — APPROVED** [review]
JS looks good to me. Once someone reviews the style, let's merge.

**Review by srartese — APPROVED** [review]
The styles look great and it resizes well with smaller screen sizes. I think keeping the width there is fine since you don't want the box to get massive with such little text.

**Pull request merged**
